infrastructure/hub-and-spoke-peerings/main.tf

```diff
@@ -0,0 +1,325 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+##############################################################
+#                              VPCs                          #
+##############################################################
+
+module "vpc-hub" {
+  source = "../../modules/net-vpc"
+
+  project_id   = var.hub_project_id
```

Comment:
Shall we use a single project? We don't really need three projects, and having three makes creating prerequisites much more complex. Lots of people have a single playground project.